adblock-lean

```diff
@@ -529,25 +539,25 @@ restore_saved_blocklist()
 
 clean_dnsmasq_dir()
 {
-	rm -f /tmp/dnsmasq.d/.blocklist.gz /tmp/dnsmasq.d/blocklist /tmp/dnsmasq.d/conf-script /tmp/dnsmasq.d/.extract_blocklist
+	rm -f /var/run/adblock-lean/dnsmasq.d/.blocklist.gz /var/run/adblock-lean/dnsmasq.d/blocklist /var/run/adblock-lean/dnsmasq.d/conf-script /var/run/adblock-lean/dnsmasq.d/.extract_blocklist
 }
```

Comment:
(another conf-script instance)

Reply:
Yes, thanks. Forgot about all the dnsmasq.d special case when I did the find and replace.